### DISCUSSION TITLE
Block a specific client from logging in for a set number of minutes after Max failed password login attempts has been reached

### DISCUSSION THREAD

**Discussion by pacman366** [rustdesk/rustdesk#1124] [General]
Ideally by default after 5 ~ 10 failed password login attempts within a period of 10 minutes of each other, reject subsequent requests for 15 ~ 30 minutes from that specific client silently. Block logon attempts using both source IP address and source client ID.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/blob/3458c164fbaa19e6be6a4ef0d1df878c8bbdf146/src/server/connection.rs#L980
